lib/py/images/images.py

```diff
@@ -415,23 +411,15 @@ def skip(self, f, pbuff):
         f.seek(round_up(size, sizeof_u64), os.SEEK_CUR)
         return size
 
+    def _get_sem_array(self):
+        s = array.array('H')
+        if s.itemsize != sizeof_u16:
+            raise Exception("Array size mismatch")
+        return s
 
 class ipc_msg_queue_handler:
```

Comment:
Would you be able to add two blank lines before every class definition?
```console
$ flake8 --select E302 --config=scripts/flake8.cfg lib/py/images/images.py
lib/py/images/images.py:373:1: E302 expected 2 blank lines, found 1
lib/py/images/images.py:387:1: E302 expected 2 blank lines, found 1
lib/py/images/images.py:420:1: E302 expected 2 blank lines, found 1
lib/py/images/images.py:466:1: E302 expected 2 blank lines, found 1
```

Reply:
Sure!

Reply:
@behouba Can you also add `lib/py/images/images.py` to the list of files `flake8` is run during CI. Have a look at the Makefile and add `lib/py/images/images.py` to where `flake8` is called.